src/components/ui/Suggest.jsx

```diff
@@ -1,27 +1,40 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect, useState, useContext } from 'react';
+import ReactDOM from 'react-dom';
 import debounce from 'lodash.debounce';
+import { bool, string, func, object } from 'prop-types';
 
 import SuggestsDropdown from 'src/components/ui/SuggestsDropdown';
 import { fetchSuggests, selectItem, modifyList } from 'src/libs/suggest';
+import { DeviceContext } from 'src/libs/device';
 
 const SUGGEST_DEBOUNCE_WAIT = 100;
 
 const Suggest = ({
-  tagSelector,
+  inputNode,
+  outputNode,
   withCategories,
   withGeoloc,
   onSelect = selectItem,
+  className,
 }) => {
   const [items, setItems] = useState([]);
   const [isOpen, setIsOpen] = useState(false);
   const [lastQuery, setLastQuery] = useState('');
-  const searchInputDomHandler = document.getElementById(tagSelector);
+  const isMobile = useContext(DeviceContext);
   let currentQuery = null;
 
   useEffect(() => {
     const handleFocus = () => {
-      setIsOpen(true);
-      fetchItems(searchInputDomHandler.value);
+      if (inputNode.value === '') {
+        setIsOpen(true);
+        fetchItems('');
+      } else {
+        inputNode.select();
+        if (isMobile) {
+          setIsOpen(true);
+          fetchItems(inputNode.value);
+        }
```

Comment:
As a consequence, it's now impossible to see the suggestions list after the search field is blurred and no result has been selected. Is it really expected ?

Reply:
you mean on desktop right ? This is the current design choice, we should contribute your comment to the discussion about design, what do you think ?